ForageSDK.podspec

```diff
@@ -1,14 +1,15 @@
 Pod::Spec.new do |spec|
 
   spec.name         = "ForageSDK"
-  spec.version      = "4.1.2"
+  spec.version      = "4.1.3"
   spec.summary      = "ForageSDK"
   spec.description  = "The ForageSDK process Electronic Benefit Transfer (EBT) payments in your e-commerce application."
   spec.homepage     = "https://github.com/teamforage/forage-ios-sdk"
   spec.license      = { :type => "MIT", :file => "LICENSE" }
   spec.author       = { "Rob Gormisky" => "rob@joinforage.com" }
   spec.platform     = :ios, "13.0"
-  spec.source       = { :git => "https://github.com/teamforage/forage-ios-sdk.git", :tag => "4.1.2" }
+  spec.readme       = "https://raw.githubusercontent.com/teamforage/forage-ios-sdk/main/README.md"
```

Comment:
This should display the up-to-date README.md instead of the out-of-date [one seen here](https://cocoapods.org/pods/ForageSDK)

- [Cocoapods docs](https://guides.cocoapods.org/syntax/podspec.html#group_subspecs:~:text=The%20URL%20for%20the%20README%20markdown%20file%20for%20this%20pod%20version.)
- https://stackoverflow.com/a/76240887

Reply:
Ahhhhh thank you sir! I noticed this a while ago and thought we were just SOL